src/common/queries/credits.ts

```diff
@@ -37,7 +37,7 @@ export const useBulk = () => {
     }).then(() => {
       const message =
         successMessages[action as keyof typeof successMessages] ||
-        `${action}d_invoice`;
+        `${action}d_credit`;
```

Comment:
@turbo124 Our tests discovered a minor bug. Instead of receiving messages for archived/deleted/restored credits, we received the same message that is used for invoices. Therefore, it's just a small change in the translation keyword.